catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/LibraryImpl.java

```diff
@@ -328,10 +354,14 @@ private Object getCatalogMgrObject(String managerId) {
   public LibraryDescription get_library_description() throws ProcessingFault, SystemFault {
     LOGGER.trace("get_library_description called");
     String host = System.getProperty(SystemBaseUrl.EXTERNAL_HOST);
-    String country = System.getProperty("user.country");
-    String organization = System.getProperty("org.codice.ddf.system.organization");
-    String libraryDescr = country + "|" + organization;
-    return new LibraryDescription(host, libraryDescr, libraryVersion);
+    String desc = libraryDescription;
+    if (StringUtils.isBlank(desc)) {
+      String country = System.getProperty("user.country");
+      String organization = System.getProperty("org.codice.ddf.system.organization");
```

Comment:
can't we use https://github.com/codice/ddf/blob/master/libs/common-system/src/main/java/org/codice/ddf/configuration/SystemInfo.java#L40?

Reply:
Yep. Changed